tests/app/main/views/test_suppliers.py

```diff
@@ -136,14 +141,14 @@ def test_declaration_and_agreement_links_visible_for_ccs_sourcing(self, role,
 
     def test_agreements_template_only_lists_frameworks_that_we_care_about_in_reverse_order(self):
         interesting_frameworks = [
-            {'slug': 'digital-mash-2', 'name': 'Digital Mash 2', 'framework_id': 4},
-            {'slug': 'digital-mash-1', 'name': 'Digital Mash 1', 'framework_id': 2},
-            {'slug': 'sausage-cloud-1', 'name': 'Sausage Cloud 1', 'framework_id': 1},
-            {'slug': 'sausage-cloud-3', 'name': 'Sausage Cloud 3', 'framework_id': 5},
-            {'slug': 'sausage-cloud-2', 'name': 'Sausage Cloud 2', 'framework_id': 3},
+            {'slug': 'digital-mash-2', 'name': 'Digital Mash 2', 'id': 4},
+            {'slug': 'digital-mash-1', 'name': 'Digital Mash 1', 'id': 2},
+            {'slug': 'sausage-cloud-1', 'name': 'Sausage Cloud 1', 'id': 1},
+            {'slug': 'sausage-cloud-3', 'name': 'Sausage Cloud 3', 'id': 5},
+            {'slug': 'sausage-cloud-2', 'name': 'Sausage Cloud 2', 'id': 3},
```

Comment:
Are these the typos you found?

Reply:
Not quite! 😹 The new stub intentionally uses `id` instead of `framework_id` to match the other models.

The typos were `framework_suffixs` -> `framework_suffixes`